src/main/java/baubles/client/ClientProxy.java

```diff
@@ -45,6 +46,6 @@ public World getClientWorld() {
 	@Override
 	public void registerItemModels() {
 		Minecraft.getMinecraft().getRenderItem().getItemModelMesher().register(Config.itemRing, 0, new ModelResourceLocation(Baubles.MODID + ":ring", "inventory"));
-		ModelBakery.addVariantName(Config.itemRing, Baubles.MODID + ":ring");
+		ModelBakery.registerItemVariants(Config.itemRing, new ResourceLocation(Baubles.MODID + ":ring"));
```

Comment:
This section is incorrect anyway, the whole method needs to be replaced with one ModelLoader.setCustomModelResourceKocation call